extensions/podman/package.json

```diff
@@ -115,6 +115,23 @@
         }
       }
     },
+    "icons": {
+      "podman-icon": {
+        "description": "Podman icon",
+        "default": {
+          "fontPath": "podman.woff2",
+          "fontCharacter": "\\0041"
+        }
+      }
+    },
+    "views": {
+      "icons/containersList": [
+        {
+          "when": "io.podman.demo in containerLabelKeys",
+          "icon": "${podman-icon}"
+        }
+      ]
+    },
```

Comment:
I'm wondering if it should be part of this extension

IMHO it's dedicated to a demo extension.

Reply:
do you think it'd be a good idea to keep it in podman/package.json as it's a good practical example of how to use this functionality for other extensions?

I feel like the first thing a developer does when checking out how to create an extensions is checking our extensions/podman/package.json and having this here would be a good example of how package.json works well with the extension.

Reply:
I can of course create a separate extension for the view contribution, it just felt minor and harmless. It felt like overkill to create an extension for 9 lines :) and I couldn't think of anything else we'd put in that extension - unless we'd want an extension to showcase sample extension code like this, or a Demo Now button that would preload a bunch of sample images & containers. I'm ok either way.

Reply:
I think having an extension https://github.com/redhat-developer/podman-desktop-demo/ would make sense to me

Reply:
I agree with @deboer-tim that having this would be harmless to include as a cute little easter egg / example of how an extension is created.

We could talk about this more in the next call. If we decide to not include it in package.json we can just "demo" the extension by importing the image into PD, etc.

Reply:
Now that we have a Podman Desktop demo extension imminent (https://github.com/redhat-developer/podman-desktop-demo/pull/27), I'm ok dropping this back to just the icon and will propose the view context there. PR updated.